Add tip on how to check that the CXI provider is working correctly (#233)
eth-cscs/cscs-docs: docs/software/container-engine/resource-hook.md

    4194304             23925.61
    ```

> !!! tip "How to check if the CXI provider is available"

Reviewer: ```suggestion
!!! tip "How to check the CXI provider inside a container"
```

    You can check if the CXI provider is available using `fi_info`

>     (within the container):

Reviewer: The `within the container` is the key context here (I think?)

Maybe start the section with something like "If you are uncertain about whether the CXI provider was configured correctly inside your running container..."
